CHANGELOG.md

```diff
@@ -11,15 +11,20 @@ Unreleased` header.
 
 # Unreleased
 
+- **Breaking:** Deprecate `EventLoop::create_window`.
+- **Breaking:** Move `Window::new` to `ActiveEventLoop::create_window` and `EventLoop::create_window`
```

Comment:
Nit:

```suggestion
- **Breaking:** Move `Window::new` to `ActiveEventLoop::create_window` and `EventLoop::create_window`. The latter is deprecated, you should instead be creating your window inside the `NewEvents(StartCause::Init)` or `Resumed` events.

    ```rust
    // Before
    let window = Window::new(&event_loop);
    event_loop.run(|event, event_loop| {
        match event {
            // Handle events
        }
    });
    
    // After
    let mut window = None;
    event_loop.run(|event, event_loop| {
        match event {
            Event::Resumed => {
                window = Some(event_loop.create_window(Window::attributes()));
            }
            Event::Suspended => {
                window = None;
            }
            // .. Handle events
        }
    });
    ```
```

Reply:
See https://github.com/rust-windowing/winit/issues/3477

Would probably send it separately though.

Reply:
> Do we really want that in the CHANGELOG?

Well, migrations are only really relevant for a particular release, so I think the changelog is the _perfect_ place to put those.